src/MC/fix_widom.cpp

```diff
@@ -1050,13 +1050,7 @@ double FixWidom::energy_full()
 
   if (force->kspace) force->kspace->compute(eflag,vflag);
 
-  // unlike Verlet, not performing a reverse_comm() or forces here
-  // b/c Widom does not care about forces
-  // don't think it will mess up energy due to any post_force() fixes
-  // but Modify::pre_reverse() is needed for INTEL
-
-  if (modify->n_pre_reverse) modify->pre_reverse(eflag,vflag);
-  if (modify->n_pre_force) modify->pre_force(vflag);
```

Comment:
Why remove pre_force? Only pre_reverse was added for INTEL package compatibility in 2e7e5aeac4b90ad13d034c34771e170863ad532e

Reply:
Why are we calling pre_force twice here (in the original code block)?

Reply:
I don't understand the above code snippet. Instead of pre_force(), it should be post_force().

Reply:
@evoyiatzis Do you have any comments on this?